### PR TITLE
DEMO-163: As a visitor I want to change my password

### PR DESCRIPTION
**Pull request by ciastektk**
https://jira.ez.no/browse/DEMO-163

Screen Shot 2019-04-12 at 10 55 05 (2)

Screen Shot 2019-04-12 at 10 55 32 (2)

### CODE REVIEW

File: src/AppBundle/Controller/PasswordController.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Templating\EngineInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class PasswordController
```
Comment (andrerom) [inline src/AppBundle/Controller/PasswordController.php]: @SylvainGuittard / @lserwatka  Shouldn't this be in user package so developers can reuse this?
ref: https://github.com/ezsystems/ezplatform-user

Comment (lserwatka) [reply src/AppBundle/Controller/PasswordController.php]: Yes, but we need to change code in the ezplatfrom-user. This is stage 2. Stage 1 was to extract code into the separate bundle.

Comment (lserwatka) [reply src/AppBundle/Controller/PasswordController.php]: Actually, this controller might not be needed here at all. @damianz5 is checking. Entire logic (exactly the same) in the ezplatfrom-user bundle.

**Comment by ciastektk** [comment]
@lserwatka, @andrerom I've found a way how to use UserBundle to change password. PR has been updated.

**Comment by andrerom** [comment]
@ciastektk Great!! For future reference, don't hesitate when working on demos/ezplatform.com/.., to propose changes to our bundes/packages on functionality meant to be re-usable. On how, just check with the others in the office for opinions.

Besides saving yourselves from maintaining duplicated code you are saving partners from having to do the same and you help make the product easier to use (DX) ;)

**Pull request merged**